nextjs/src/components/cart/index.tsx

```diff
@@ -0,0 +1,139 @@
+                      <div key={product.id} className={styles.productContainer}>
+                        <div className={styles.product}>
+                          <figure className={styles.productImage}>
+                            <Image
+                              width={150}
+                              height={150}
+                              alt="shoe"
+                              src={product.imgSrc}
+                            ></Image>
+                          </figure>
+                          <div className={styles.productInfo}>
+                            <div className={styles.productDetails}>
+                              <span>{product.name}</span>
+                              <div className={styles.detail}>
+                                {product.category}
+                              </div>
+                              <div className={styles.detail}>
+                                {product.variant}
+                              </div>
+                              <div className={styles.detail}>
+                                {product.size}, Quantity{" "}
+                                {product.quantityInCart}
+                              </div>
+                            </div>
+                            <div className={styles.productPrice}>
+                              <span>
+                                ${product.price * (product.quantityInCart || 1)}
+                              </span>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    ))
+                  : "No products in your cart"}
+              </div>
+            </div>
+            <div className={styles.paymentCol}>
+              <aside className={styles.paymentContainer}>
+                <h4>Overview</h4>
+                <div className={styles.paymentDetailContainer}>
+                  Subtotal
+                  <div className={styles.paymentDetailSummary}>
+                    ${totalAmount.toFixed(2)}
+                  </div>
+                </div>
+                <div className={styles.paymentDetailContainer}>
+                  Shipping cost
+                  <div className={styles.paymentDetailSummary}>Free</div>
+                </div>
+                <div className={styles.totalContainer}>
+                  <span>Total</span>
+                  <span className={styles.total}>
+                    ${totalAmount.toFixed(2)}
+                  </span>
+                </div>
+                <div className={styles.actions}>
+                  <section>
+                    <button
+                      className={styles.checkoutButton}
+                      type="submit"
+                      role="link"
+                      onClick={handleCheckout}
+                      disabled={!cartProducts.length}
+                    >
+                      Checkout
```

Comment:
This just links to an undefined page - should probably just go to the home page? All the tracking works though

Reply:
I think this is because the Stripe envvars need to be set to be able to access the checkout. I'll make it a bit clearer in the README.